### ISSUE TITLE
Studies missing associated data sets

### ISSUE THREAD

**Issue by ssarrafan** [microbiomedata/nmdc-server#877]
Montana discovered through a search that there are studies on the data portal without associated data sets. She would like to find out if there are others besides the example below so that we can follow up on any missing data. 

Screenshots show the sample search which finds 116 samples but only 110 metagenome data sets. 
<img width="1476" alt="Screen Shot 2023-03-02 at 1 11 39 PM" src="https://user-images.githubusercontent.com/79225425/222556398-aa28061c-6e8d-4937-81a3-94bf844ff25b.png">

missing data sets?
<img width="1147" alt="Screen Shot 2023-03-02 at 1 13 01 PM" src="https://user-images.githubusercontent.com/79225425/222556418-c5ffee23-7f5c-41a5-82a0-739c45e2e697.png">

FYI @mslarae13 @aclum 

Adding this to the backlog as it's not urgent and Jeff said this could take some time to find.

**Comment by aclum** [comment]
It is worth noting that these samples are from bioscales and we are replacing all the records for GSP https://github.com/microbiomedata/issues/issues/57

This should be discussed in the context of re-iding vs reprocessing, that is we may want to wait until after we re-process to avoid having to find these edge cases twice

**Comment by mslarae13** [comment]
It's been discussed but I think this is cuz there are biosamples that don't have data?

**Comment by aclum** [comment]
Yes, there can be a different number of records b/c one is a count of biosamples (biosample_set) and the other is a count of omics processing records (omics_processing_set). The counts are off currently with the reload b/c we made biosamples for the metabolomics only biosamples but did not make omics processing set records, instead we just linked to the raw data at the study level. It is expected that these won't necessarily match. Another example is NEON soils samples where there will be approximately 3 biosamles: 1 omics_processing_set record of type 'metagenome'. I do think we should make it clearer that 'Found x results' in the top left refers to biosamples so I'll spin that out to a new request.
